warehouse/query-core/src/main/java/datawave/query/predicate/EmptyDocumentFilter.java

```diff
@@ -21,7 +21,7 @@ public class EmptyDocumentFilter implements Predicate<Entry<Key,Document>> {
     
     @Override
     public boolean apply(Entry<Key,Document> input) {
-        boolean nonempty = (input.getValue().size() > 0);
+        boolean nonempty = (!input.getValue().isEmpty() || input.getValue().hasTimingMetadata());
```

Comment:
I am not picking up the significance of having timing metadata, and how that changes this operation, but it's also late on a friday.  Can you explain a bit?

Reply:
Previously the TimeingMetadata was a Document that was being added to the dict making the document non-empy.  Basically it is valid to have a document with only timing metadata and that should not be considered an empty document in this case.